_redirects

```diff
@@ -15,10 +15,12 @@
 # ----------
 # GitHub: https://github.com/apollo-specs/core
 # Netlify: https://app.netlify.com/sites/apollo-specs-core/
+/core/ /core/v0.1/ 302!
```

Comment:
Ah good point on 302 vs 301.

Does ! mean proxy instead of redirect?

Reply:
The 200 status code is the designator that indicates "proxy". The exclamation means "force", which means "don't shadow".

Netlify talks about Shadowing here:

https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing

Essentially I think that means that if a file exists in path on the local deployment site (e.g., a file called _`core`_ on the `specs` in this case), don't use that as a reason _not_ to redirect - redirect anyway.  That seems defensive to me given our current needs.

Reply:
And in this case I thought the redirect was better than the proxy.  It will proxy to the separate deployment of the docs once it hits the redirect though!

Reply:
Yep, I agree that redirect is better and just wanted to make sure that was what it was!

Do you have a mnemonic on 301 vs 302 other than looking it up every time? I meant temporary and said the wrong one :)

Reply:
**T**wo is for **t**emporary!